openpype/plugins/publish/integrate.py

```diff
@@ -795,9 +797,15 @@ def prepare_representation(self, repre,
 
             # Multiple file transfers
             transfers = []
-            for src_file_name, dst in zip(src_collection, dst_collection):
+            filtered_files = files.copy()
+
+            for src_file_name, dst, index, file in zip(src_collection, dst_collection, destination_indexes, files):
+                if custom_frames and index not in custom_frames:
```

Comment:
`if index not in custom_frames`

Reply:
`custom_frames` est toujours une liste (vide ou non) donc il est plus simple de simplement checker si ton index est dedans, le python est assez optimisé pour skip si la liste est vide

Reply:
oui, mais si jamais je veux publish sans avoir donné de custom_frame, tous mes files vont être remove et donc rien ne sera intégré

Reply:
Je vais y refléchir

Reply:
Apparement, si la liste est vide, il concidère que aucun index n'est dedans, et donc passe bien dans la boucle... Ce qui remove tous mes file de ma liste à intégrer ^^'

Je vais donc laisser la double verification pour l'instant

Reply:
L'entièreté de ce code n'est pas nécessaire car les réprésentations doivent déjà être filtré avant d'arriver dans le integrate.py

Reply:
Ce n'est en effet pas le cas pour le moment
Tu suggère de les filtrer avant d'arriver ici ? 
Je pense que c'est faisable, mais le soucis que j'ai c'est que le passage par ffmpeg necessite des sequances d'image ininterrompu pour fonctionner, de même que les fonctions de clique.assemble.

Je vais essayer de regarder les possibilités, je reviendrai mettre un commentaire si necessaire